### ISSUE TITLE
Gitter channel for discussion

### ISSUE THREAD

**Issue by Mordax** [0xazure/supernova#14]
For anyone new joining the repo, here's the gitter link: 
https://gitter.im/supernova-rust/Lobby

supernova was taken :/

**Comment by 0xazure** [comment]
Hey @Mordax I think this is a really neat idea 💬 to give people many different ways to interact with the project.

Is there a way to link this channel to the repository? For example, does it integrate with GitHub issues/pull requests and display them in the rooms?

I've also seen some shiny Gitter badges on some projects' README, how can we get one of those for this project?

**Comment by Mordax** [comment]
Yes, we can integrate GH issues and pulls with Gitter. We can add Travis as well. 
Repo owner has to link to the Gitter though because I don't own the repo to do it. And we can do permission control as well. 
![config](https://user-images.githubusercontent.com/9853904/48288454-31d3cb80-e43a-11e8-8da1-9d62ef682990.PNG)
Copy-able token: 8a33ff5a1d30204f600c.

You can add gitter chat like this:
`[![Gitter chat](https://badges.gitter.im/supernova-rust/Lobby.png)](https://gitter.im/supernova-rust/Lobby)`
